test/extended/operators/tolerations.go

```diff
@@ -32,6 +32,8 @@ var _ = Describe("[sig-arch] Managed cluster should", func() {
 		excludedNamespaces := sets.NewString("openshift-kube-apiserver", "openshift-kube-controller-manager", "openshift-kube-scheduler", "openshift-etcd", "openshift-openstack-infra", "openshift-ovirt-infra")
 		// exclude these pods from checks
 		whitelistPods := sets.NewString("network-operator", "dns-operator", "olm-operators", "gcp-routes-controller", "ovnkube-master", "must-gather")
+		// The kube-apiserver proxy exists only in Hypershift
```

Comment:
please give this a full comment on why it is being whitelisted (e.g. the summary from your PR)

Reply:
Done